README.md

```diff
@@ -10,6 +10,10 @@ This will start a local server and can be accessed at http://localhost:8080/blog
 1. Run `npm install`
 2. Run `npm run start:blog`
 
+## Troubleshoot
+If you see errors after running `npm install`, please double check if you nodejs version matches the one in `package.json`. We will perform regular upgrade on node version we use in
+our project. Thanks.
```

Comment:
I'm not so sure we'll do _regular_ upgrades, but yea, I suppose :)
```suggestion
If you see errors after running `npm install`, please double check if you nodejs version matches the one in `package.json`. We will perform regular upgrade on node version we use in
our project.
```

Reply:
and "thanks" just reads kinda awkwardly there imo

Reply:
Will delete it in next commit